examples/check_unitary_std.py

```diff
@@ -38,16 +57,18 @@
 print("These values are NOT the same so we need a unitary FT for this to work")
 example_data.ft('t', shift=True, unitary=True)
 example_data.ft('ph', unitary=True)
-print("When we apply a unitary FT the std of the frequency is:",
-        sqrt(example_data.C.run(np.var,'t').mean('ph')/2))
+freq_dom_std = sqrt(example_data.C.run(np.var,'t').mean('ph')/2)
+print("When we apply a unitary FT the std over all the frequency domain is:",
+        freq_dom_std)
+print("Because we have no signal, this again corresponds to our noise.")
```

Comment:
I'm converting some commentary to print statements -- but also, watch the fold markers.  The comment at the top of the fold marker should describe **everything** inside the fold marker.